src/Roave/InfectionStaticAnalysis/CliUtility.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\InfectionStaticAnalysis;
+
+use function array_values;
+use function strpos;
+use function substr;
+
+/**
+ * @internal
+ */
+final class CliUtility
+{
+    private function __construct()
+    {
+    }
+
+    /**
+     * @return list<non-empty-string>
+     */
+    public static function getArguments(): array
+    {
+        /** @var list<non-empty-string> */
+        return $_SERVER['argv'] ?? [];
```

Comment:
Can we keep this in the binary, and have this utility only modify the `$_SERVER` values? 🤔